### ISSUE TITLE
Migrations for microsoft_id field are not synchronized with models.py

### ISSUE THREAD

**Issue by zen4ever** [AngellusMortis/django_microsoft_auth#184]
* Django Microsoft Authentication Backend version: 1.2.0
* Python version: 3.6
* Operating System: Ubuntu 16.04

### Description

Migrations file has microsoft_id field as max_length 32 and models.py have it as max_length=36.
I can't login with Office365 account because it says microsoft_id column is not big enough. 

### What I Did

```
$ ./manage.py makemigrations
Migrations for 'microsoft_auth':
  /usr/local/lib/python3.6/site-packages/microsoft_auth/migrations/0002_auto_20190228_2006.py
    - Alter field microsoft_id on microsoftaccount
$ ./manage.py migrate
Operations to perform:
  Apply all migrations: admin, auth, catalog, contenttypes, microsoft_auth, risk_management, sessions, sites
Running migrations:
  Applying microsoft_auth.0002_auto_20190228_2006... OK
```
this fixes the issue, but would be better if migration was in the repo in the first place

**Comment by zen4ever** [comment]
Below is migration that gets generated

```python
# Generated by Django 2.1.7 on 2019-02-28 20:06

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('microsoft_auth', '0001_initial'),
    ]

    operations = [
        migrations.AlterField(
            model_name='microsoftaccount',
            name='microsoft_id',
            field=models.CharField(max_length=36, verbose_name='microsoft account id'),
        ),
    ]
```

**Comment by AngellusMortis** [comment]
Fixed in `1.2.1` (just pushed it). Sorry that migration seemed to have gotten missed when the number was updated from 32 to 36 and I never noticed since my `microsoft_id` is less than 32.
